src/App.js

```diff
@@ -1,18 +1,37 @@
 /** @jsx jsx */
 import { jsx } from 'theme-ui';
-
+import taskContext from './context/task_context';
+import SignUpForm from './components/SignupForm';
 import Header from './components/header/Header';
 import PrivateRoute from './PrivateRoute';
 import Dashboard from './components/dashboard/Dashboard';
+import axios from 'axios';
+import { Route } from 'react-router-dom';
+
+
+
+const App = (props) => {
+
+  const SignUp = (prop) => {
```

Comment:
We'll probably want to abstract out out context methods and state to a separate file as we add more

Reply:
100 percents. more just getting stuff working at this point. i think the entire app structure should eventually be changed